test/jdk/java/net/httpclient/HttpClientBuilderTest.java

```diff
@@ -271,6 +271,34 @@ public void testSSLParameters() {
         try (var closer = closeable(builder)) {
             assertTrue(closer.build().sslParameters().getProtocols()[0].equals("C"));
         }
+        // test defaults for needClientAuth and wantClientAuth
+        builder.sslParameters(new SSLParameters());
+        try (var closer = closeable(builder)) {
+            assertFalse(closer.build().sslParameters().getNeedClientAuth(),
+                    "needClientAuth() was expected to be false");
+            assertFalse(closer.build().sslParameters().getWantClientAuth(),
+                    "wantClientAuth() was expected to be false");
+        }
+        // needClientAuth = true and thus wantClientAuth = false
+        final SSLParameters needClientAuthParams = new SSLParameters();
```

Comment:
You may also want to remove `final` at this line and line 293, though this is just be a test.

Reply:
Done. I've updated the PR to remove the introduction of `final`.